src/CMakeLists.txt~

```diff
@@ -0,0 +1,128 @@
+cmake_minimum_required(VERSION 2.8)
```

Comment:
Not sure if I understand what you mean by a bad copy? That it shows like the whole file is new, when in fact it's not?